Tests/FunctionalTests/PluginTests.swift

```diff
@@ -299,13 +299,18 @@ final class PluginTests {
 
     @Test(
         .enabled(if: (try? UserToolchain.default)!.supportsSwiftConcurrency(), "skipping because test environment doesn't support concurrency"),
-        .enabled(if: ProcessInfo.hostOperatingSystem == .macOS, "Test is only supported on macOS")
+        .enabled(if: ProcessInfo.hostOperatingSystem == .macOS, "Test is only supported on macOS"),
+        arguments: [BuildSystemProvider.Kind.native, .swiftbuild]
     )
-    func testUseOfVendedBinaryTool() async throws {
+    func testUseOfVendedBinaryTool(buildSystem: BuildSystemProvider.Kind) async throws {
```

Comment:
This comment accompanies the one suggesting build against all build configurations.
```suggestion
    func testUseOfVendedBinaryTool(buildSystem: BuildSystemProvider.Kind, configuration: BuildConfiguration) async throws {
```